inference-engine/src/mkldnn_plugin/nodes/mkldnn_space_to_batch_node.h

```diff
@@ -22,11 +22,12 @@ class MKLDNNSpaceToBatchNode : public MKLDNNNode {
     void execute(mkldnn::stream strm) override;
     bool created() const override;
 
+    bool needPrepareParams() const override { return false; };
```

Comment:
as I can see we have separate PR for `space_to_batch` https://github.com/openvinotoolkit/openvino/pull/8701
why do we need this changes here?

Reply:
It should be caused by my mistake cherry-pick when applying review comments. Already reverted it.